### ISSUE TITLE
Questions about data sets and indicators

### ISSUE THREAD

**Issue by ChaohuanDeng123** [NITR098/Awesome-U-Net#9]
Hello, thank you for doing such a excellent job！I'm a little confused about this data set and metrics.
Can you help me answer them?
First, as far as I know, the data set of ISIC Task 2018 officially has validation set and test set. May I ask why the training set should be separately divided into training set, validation set and test set? Instead of using the official validation set and test set?
Second question, in this paper, do the relevant indicators in Table 2 (a) refer to the validation set or the test set? I have downloaded your pre-training weights and found that they do not match the results in the paper.

**Comment by anran0422** [comment]
@ChaohuanDeng123 论文中的结果，很可能是最好的测试集中的结果。

与论文中结果不符是不是体现在miou f1 sen上
